scripts/generate-google3-docs/resource-reference/_toc.yaml

```diff
@@ -525,3 +525,7 @@ toc:
   section:
   - title: "VPCAccessConnector"
     path: /config-connector/docs/reference/resource-docs/vpcaccess/vpcaccessconnector.md
+- title: "Vertex AI"
```

Comment:
Could you move the section up to be before "VPC Access" section? I think we prefer to for the alphabetical order but not the ASCII char order.

Reply:
Done. Is the ordering enforced? I don't remember seeing it in the new resource guide.

Reply:
Good catch! Could you also do a quick update of the doc? Would be nice if we have tests to automate it, but that's not a priority (b/315184931).